go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prebid/prebid-server/v3
 
-go 1.23
+go 1.23.0
```

Comment:
It appears Go now requires the full version, as there was an error in downloading Go with only the major and minor versions specified.